src/Core/src/Handlers/GraphicsView/GraphicsViewHandler.Windows.cs

```diff
@@ -24,6 +24,14 @@ private protected override void OnDisconnectHandler(FrameworkElement platformVie
 
 			platformView.Loaded -= OnLoaded;
 		}
+		
+		public static void MapBackground(IGraphicsViewHandler handler, IGraphicsView graphicsView)
+		{
+			if (graphicsView.Background is not null)
+            {
+                handler.PlatformView?.Invalidate();
+            }
+		}
```

Comment:
You can make them internal and don't modify the PublicAPI I think

Reply:
Thank you for reviewing my PR! In response to your feedback, we have updated the modifiers from public to internal.

Reply:
Cool! Thanks :)

Reply:
Can we add a // TODO to make it public?